manage/sawtooth_manage/docker.py

```diff
@@ -315,3 +315,8 @@ def is_running(self, node_name):
             if node_name == entry.name:
                 return entry.status.startswith("Up")
         return False
+
+
+class CliException(Exception):
```

Comment:
This should be named something specific to manage, since this is not in the CLI subproject